### ISSUE TITLE
Being able to group fields inside a model

### ISSUE THREAD

**Issue by stefanoverna** [stefanoverna/wishlist#13]
Is it possible to group the fields in a model?  Currently it’s just a list with fields. Would be nice to give editors a bit more info how fields are related to another and make the UI for them easier to understand.

-----

Originally posted by info@cantierecreativo.net

**Comment by stefanoverna** [comment]
Maybe the same way as prismic, they use tabs  
   
 

-----

Originally posted by jan@digitaler-spinat.com

**Comment by stefanoverna** [comment]
Yes I double thumbs up this idea 
 

-----

Originally posted by experts@parkfieldcommerce.com

**Comment by stefanoverna** [comment]
Yes please! – with a way to then collapse the group or even start it in a collapsed state. Tabs another good approach. 
 

-----

Originally posted by webmaster@convincible.media

**Comment by stefanoverna** [comment]
Just an idea 
   
 Vertical tabs for the models 
 Horizontal tabs for the different languages 
   
 Or reverse 
   
   
 

-----

Originally posted by jan@digitaler-spinat.com

**Comment by matjack1** [comment]
This feature request has been moved to https://community.datocms.com/t/357
